css/style.css

```diff
@@ -46,12 +46,52 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
-     color: #000;
+     color: #fee;
      font-size: 12px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+     font-family: "open sans";
+     background: url(../images/header_bg.jpg) repeat top rgb(134,141,157);
```

Comment:
:hammer: Don't forget our conversation about setting your html up first... good structure will lead to good css later. This image should only be on the "top" section of the page. You would need to re-think you're structure a bit to make it work.